src/settings.ts

```diff
@@ -135,7 +178,64 @@ export class WikipediaSearchSettingTab extends PluginSettingTab {
 								settings.templates[i].name = value;
 								await this.plugin.saveSettings();
 							})
-					)
+					);
+				
+				new Setting(containerEl)
+					.setName(`Create Note`)
+					.setDesc(`Enable to create a new note for the article, instead of just linking it. By activating this option, the template will change to a preferred setup. After activating you can change the template for your needs.`)
+					.addToggle((toggle) =>
+						toggle
+							.setValue(val.createArticleNote)
+							.onChange(async (value) => {
+								settings.templates[i].createArticleNote = value;
+								if (settings.templates[i].createArticleNote) {
+									settings.templates[i].templateString = "{thumbnail}\n{intro}\n\n{url}";
+								} else {
+									settings.templates[i].templateString = DEFAULT_SETTINGS.defaultTemplate;
+								}
+								await this.plugin.saveSettings();
+								this.display();
+							})
```

Comment:
I don't like the idea that it would just instantly override any previous template. Maybe only change the template to `{thumbnail}\n{intro}\n\n{url}` or something if the old template is the default one.

Reply:
should I then leave it in so that when I turn off the Create Note function, the template when it was set to the one I still set back to the default template or should that then just stay as it is?

Reply:
Stay as is please :)